src/levels/level_1.gd

```diff
@@ -20,3 +20,4 @@ func puzzle_completed() -> void:
 	$Player.unfreeze()
 	$PuzzleConsole.hide()
 	$Door.open()
+	$Terminal.turn_on()
```

Comment:
Should some of these things be in a super level puzzle_completed function?

Reply:
Agreed it would be nice to hook things together, but it is possible that we have multiple puzzle completions and it isn't guaranteed that each one will have the same output (doors vs getting an unlock etc) so I think having a different function for each is pretty much required